x/staging/posts/keeper/msgs_server.go

```diff
@@ -300,25 +299,25 @@ func (k msgServer) AnswerPoll(goCtx context.Context, msg *types.MsgAnswerPoll) (
 
 	// Make sure the post has a poll
 	if post.Poll == nil {
-		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest,
+		return nil, sdkerrors.Wrapf(types.ErrPollNotFound,
 			"no poll associated with ID: %s", msg.PostID)
 	}
 
 	// Make sure the poll is not closed
 	if post.Poll.EndDate.Before(ctx.BlockTime()) {
-		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest,
+		return nil, sdkerrors.Wrapf(types.ErrPollClosed,
```

Comment:
This should be an `sdkerrors.ErrInvalidRequest` since it's related directly to the request